### PR TITLE
sign in with different wallets popup

### LINKED ISSUE #15: Profile: Signing in
@rymksotnini - hi there!

I noticed that on https://yearn.finance/ that there is a sign in method that allows the user to access the platform with multiple wallets. When you have some time tomorrow - can you look into the following: 

1. Are they using a library here to import all of the wallets: https://yearn.finance/; I think I found something here: https://github.com/iearn-finance/iearn-finance/blob/3874a7b1f9b0d7abe996dbf8dfa0765d6da1795b/src/components/unlock/unlock.jsx... doesn't seem like it's a library
2. Is it possible to do something similar for the Bounties platform? I understand right now we can only use MetaMask... what effort is required to get something like the Yearn Finance wallet connect implementation

Bonus: Seems like https://uniswap.exchange/ also did something similar. See some of the code here: https://github.com/Uniswap/uniswap-interface/blob/8fd894f2d1b03921c5519c050aed164343c47fb1/src/components/WalletModal/index.tsx. You may have to trace through a bit more.

### PR DESCRIPTION
**Pull request by rymksotnini**
resolves #15

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -23,7 +38,7 @@
     "react-dom": "^16.13.1",
     "react-markdown": "^5.0.3",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "3.4.1",
+    "react-scripts": "^4.0.3",
```
Comment (tenthirtyone) [inline client/package.json]: What's up here? Please put back to 3.4.1

**Comment by tenthirtyone** [comment]
@rymksotnini 
On startup, is there additional configuration change?:
image

**Comment by rymksotnini** [comment]
> @rymksotnini
> On startup, is there additional configuration change?:
> image

When pushing I forgot that the .env is listed in the .gitignore. So just adding the .env file in client folder will resolve the problem

File: client/package.json (hunk 1 above)
Comment (rymksotnini) [reply client/package.json]: ok

**Comment by tenthirtyone** [comment]
:+1: Good job

Confirming using Injected method mimics existing behavior. I was able to launch/fulfill. 

@rymksotnini @mehranhydary How should we test additional login methods? I'm thinking at least the Trezor. Maybe we can task the opensource devs with testing logging in and launching a bounty?

@rymksotnini because we have the opensource devs working on issues let's make the following updates before the merge:

1. Update the README in the top-level repo with the client .env instructions.
2. Add a .env.example to the client folder with empty values for the new env vars.

After we merge I'll let the slack channel know to update their forks and merge up and give them instructions to use Injected.

**Pull request merged**